pkg/testutil/env/path.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package env
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// ExpandPath expands a relative path to a workspace specific one.
+func ExpandPath(path ...string) string {
+	if v, ok := os.LookupEnv("RUNFILES_DIR"); ok {
+		prefix := []string{v, os.Getenv("TEST_WORKSPACE"), os.Getenv("BUILD_WORKSPACE_DIRECTORY")}
+		return filepath.Join(append(prefix, path...)...)
+	}
+
+	// when not running with bazel
+	return filepath.Join(append([]string{ProjectDir()}, path...)...)
+}
+
+// PrependPath prepends the supplied path to PATH
+func PrependPath(path ...string) {
```

Comment:
nit: rename either `ExpandPath` or `PrependPath`? having both as-is is a little misleading since one applies to the PATH env var and one doesn't

Reply:
Did a quick `s/PrependPath/PrependToPath/g`. Does that seem more clear?